packages/waku/src/router/create-pages.ts

```diff
@@ -9,6 +9,7 @@ import {
   getPathMapping,
 } from '../lib/utils/path.js';
 import type { PathSpec } from '../lib/utils/path.js';
+import type { defineEntries } from 'waku/server';
```

Comment:
Prefer relative paths when possible:
```suggestion
import type { defineEntries } from '../server.js';
```